docs/RC/RC-API.md

```diff
@@ -31,8 +31,8 @@ class Test_RemoteConfigSequence:
 
     def test_asm_switch_on_switch_off():
         # first check that both config state are ok, otherwise, next assertions will fail with cryptic messages
-        assert self.config_state_activation["apply_state"] == remote_config.ApplyState.ACKNOWLEDGED, self.config_state_activation
-        assert self.config_state_deactivation["apply_state"] == remote_config.ApplyState.ACKNOWLEDGED, self.config_state_deactivation
+        assert self.config_state_activation["asm_features_activation"]["apply_state"] == remote_config.ApplyState.ACKNOWLEDGED, self.config_state_activation
```

Comment:
devil is in the details :) rename `config_state_activation` to `config_states_activation`

```suggestion
        assert self.config_states_activation["asm_features_activation"]["apply_state"] == remote_config.ApplyState.ACKNOWLEDGED, self.config_state_activation
```

Reply:
Yep thanks!, changed it for the deactivation too :smile: